finalPrep/modZeroHero.js

```diff
@@ -56,16 +92,48 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
 
+var scaryMonster = {name: "Chupacabra",
```

Comment:
This would be easier to read if the key/value pairs were formatted on their own lines.